### PR TITLE
Branch damian esteban add new operations and models to care ops sdk cu 86a24egka

### PR DESCRIPTION
**Pull request by damianesteban**
This adds the payer resource to the SDK.

### CODE REVIEW

**Comment by damianesteban** [comment]
Task linked: [CU-86a24egka Add new operations and models to CareOps SDK](https://app.clickup.com/t/86a24egka)

File: src/resources/Payer.ts [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+import { SourceRequestOptions } from '../SourceClient'
+
+export enum PayerStatus {
+  ACTIVE = 'active',
+  INACTIVE = 'inactive',
+}
+
+export type PayersListParamsSort = 'created_at' | 'name' | '-created_at' | '-name'
+
+export interface Payer {
```
Comment (blangley28) [inline src/resources/Payer.ts:11]: Should this have description or was it left off for a reason?

File: src/resources/Payer.ts [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  /**
+   * Description for this payer. The description is not displayed and is used to
+   * capture administrative notes about the payer.
+   */
+  description?: string | null
+
+  /**
+   * The status of the payer.
+   */
+  status?: PayerStatus | null
```
Comment (blangley28) [inline src/resources/Payer.ts:124]: Does the `| null` here get treated as undefined and just not updated or is it supposed to set it to null. I think status was a required field

**Review by blangley28 — CHANGES_REQUESTED** [review]
Two minor comments for discussion

File: src/resources/Payer.ts (hunk 1 above)
Comment (damianesteban) [reply src/resources/Payer.ts:11]: it should have been added.

File: src/resources/Payer.ts (hunk 2 above)
Comment (damianesteban) [reply src/resources/Payer.ts:124]: Good point. I think it should just be optional.

**Review by blangley28 — APPROVED** [review]
LGTM

**Pull request merged**